README.md

```diff
@@ -63,7 +66,7 @@ conda activate herbie
 
 Alternatively, Herbie is published on PyPI and you can install it with pip, _but_ it requires some dependencies that you will have to install yourself:
 
-- Python 3.8, 3.9, or **3.10** (recommended)
+- Python 3.8 to 3.11
```

Comment:
I see that you designated Python 3.10 to be the "default" version supported by Herbie, but I think it is safe to bump that to Python 3.11.